### PR TITLE
Morty upgrade on 1.X

### PR DESCRIPTION
**Pull request by floion**

### CODE REVIEW

File: layers/meta-resin-qemu/recipes-connectivity/bind/bind_%.bbappend [hunk 1]
```diff
@@ -1 +0,0 @@
-BBCLASSEXTEND = "native"
```
Comment (agherzan) [inline layers/meta-resin-qemu/recipes-connectivity/bind/bind_%.bbappend]: It would be nice to have a line where it says why it's no longer needed.

**Comment by floion** [comment]
@resin-jenkins retest this please

**Comment by floion** [comment]
@resin-jenkins retest this please

**Comment by floion** [comment]
@resin-jenkins retest this please

**Comment by agherzan** [comment]
This needs bump of meta-resin too.

**Comment by floion** [comment]
Superseded by https://github.com/resin-os/resin-qemu/pull/69

**Pull request closed without merging**